app/code/Magento/Cms/Controller/Adminhtml/Block/Delete.php

```diff
@@ -9,6 +9,20 @@
 class Delete extends \Magento\Cms\Controller\Adminhtml\Block
 {
     /**
+    * Factory to access block delete methods
+    * 
+    * @var \Magento\Cms\Model\BlockFactory
+    */
+    protected $cmsBlockFactory;
+
+    public function __construct(\Magento\Backend\App\Action\Context $context, \Magento\Framework\Registry $coreRegistry,
+        \Magento\Cms\Model\BlockFactory $cmsBlockFactory)
+    {
+        parent::__construct($context, $coreRegistry);
+        $this->cmsBlockFactory = $cmsBlockFactory;        
+    }
+    
```

Comment:
See static tests report: https://travis-ci.org/magento/magento2/jobs/133509578

https://github.com/magento/magento2/pull/4726#issuecomment-222016695 seems PSR-2 compatible, PHPDoc is missing for constructor also and for property PHPDoc asterisks should be one space rigther.

Would be nice if you squash all three commits into one and force push as there is only one logical change.